Andarist/typegen types (#2674)
statelyai/xstate: .changeset/red-ants-suck.md

---
'@xstate/react': major
---

author: @Andarist
author: @mattpocock

> To avoid breaking any consumers and to leverage the newly introduced typegen support the major version of this package had to be bumped. While you can still use it with older versions of TS the typegen support in this package required at least TS 4.0.

Reviewer: ```suggestion
To avoid breaking any consumers and to leverage the newly introduced typegen support, the major version of this package had to be bumped. While you can still use it with older versions of TS, the typegen support in this package requires TS version 4.0 or greater.
```
